src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -497,4 +497,7 @@
         Items="@(CorrelationFile)"
         Overwrite="true" />
   </Target>
+
+  <!-- Setup Dumpling service to collect and upload any crash dumps -->
+  <Import Project="$(MSBuildThisFileDirectory)Dumpling.targets" Condition="'$(EnableDumpling)' == 'true'" />
```

Comment:
This change will do nothing since this targets file gets imported only by post.builds, and by that time the `RunTestsForProject` target will have run already so your `SetupDumpling` target will never be called. Instead, we can either change the official build definitions such that they pass `/p:EnableDumpling=true` OR you could also change the condition in your dumpling target to be something like:
```xml
<Target Name="SetupDumpling"
          BeforeTargets="RunTestsForProject"
          Condition="'$(TargetOS)'!='Windows_NT' Or '$(EnableCloudTest)' == 'true'">
```
since `EnableCloudTest` will be true only on Official builds.